src/common/util/firebase.ts

```diff
@@ -170,6 +179,25 @@ export async function getCardById(cardId: string): Promise<w.CardInStore> {
   return snapshot.val() as w.CardInStore;
 }
 
+export async function mostRecentCards(uid: w.UserId | undefined, limit: number): Promise<w.CardInStore[]> {
```

Comment:
I feel like if user is optional here and limit isn't, the user ID should go last? But also its kinda odd that limit isn't optional

Reply:
I like the current order, but I'll make `uid` of type `w.UserId | null` to make it more clear that it's a deliberate choice to pass in a `null` parameter.